lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -60,28 +60,34 @@ func (s *builderServicer) Build(ctx context.Context, request *builder_protos.Bui
 
 	network, err := (configurator.Network{}).FromProto(request.Network, serdes.Network)
 	if err != nil {
+		glog.V(4).Infof("LTE mconfig not build for %s: Network cast failed", request.GatewayId)
```

Comment:
glog.V(4).Infof("LTE mconfig conversion error '%v' for gateway: %s", err, request.GatewayId)

Reply:
why is this required? `GatewayId` is a string